java/se/sics/mspsim/core/USCI.java

```diff
@@ -470,7 +470,7 @@ public void byteReceived(int b) {
       //System.out.println(getName() + " byte received: " + b + " enabled:" + rxEnabled);
       if (!rxEnabled) return;
 
-    if (DEBUG || true) {
+    if (true) {
```

Comment:
This looks like a temporary forced enable debug output that should not have been committed. I think the check probably should be if `DEBUG` rather than if `true`.

Reply:
Fixed.